.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # CODEOWNERS
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-* @dotdc
\ No newline at end of file
+* @wireapp/platform-engineering @wireapp/customerops
```

Comment:
GitHub shows this error:

```
Unknown owner on line 3: make sure the team @wireapp/platform-engineering exists, is publicly visible, and has write access to the repository
* @wireapp/platform-engineering @wireapp/customerops
Unknown owner on line 3: make sure the team @wireapp/customerops exists, is publicly visible, and has write access to the repository
…@wireapp/platform-engineering @wireapp/customerops
```

Is this because we only created the teams after pushing this file?

Reply:
@lwille I do not have write permissions to this repo, can you please add both of these teams to this repo with Write permission, that would solve the issue